source/_integrations/ezviz.markdown

```diff
@@ -23,6 +23,8 @@ It also exposes an RTSP stream, by using the local camera IPs (so the device hos
 
 As there is no official documentation for the API, the component retrieves data from the API used in the EZVIZ mobile app, [hosted here](https://apiieu.ezvizlife.com).
 
+The authentication requires an EZVIZ account with two-step verification disabled. Google, Facebook, TikTok or other Oauth based accounts will not work.
```

Comment:
This line feels a bit of scope, here in the introduction of the integration.

Maybe add a known issue and troubleshooting section at the bottom of the page?